cfgov/scripts/migrate_mega_menu.py

```diff
@@ -0,0 +1,883 @@
+                                'external_link':
+                                    '/ask-cfpb/category-payday-loans/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text': 'Prepaid Cards',
+                                'external_link':
+                                    '/consumer-tools/prepaid-cards/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text': 'Student Loans',
+                                'external_link':
+                                    '/consumer-tools/student-loans/'
+                            },
+                            'nav_items': []
+                        },
+
+                    ]
+                }
+            },
+            {
+                'type': 'nav_group',
+                'value': {
+                    'group_title': 'Guides',
+                    'nav_items': [
+                        {
+                            'link': {
+                                'link_text': 'Getting an Auto Loan',
+                                'external_link':
+                                    '/consumer-tools/getting-an-auto-loan/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text': 'Managing Someone Else\'s Money',
+                                'external_link':
+                                    '/consumer-tools/' +
+                                    'managing-someone-elses-money/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text': 'Money as You Grow',
+                                'external_link':
+                                    '/consumer-tools/money-as-you-grow/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text':
+                                    'Navigating the Military ' +
+                                    ' Financial Lifecycle',
+                                'external_link':
+                                    '/consumer-tools/military' +
+                                    '-financial-lifecycle/'
+                            },
+                            'nav_items': []
+                        },
+                        {
+                            'link': {
+                                'link_text': 'Owning a Home',
```

Comment:
Same comment as above re: "Buying a House".

Reply:
Link text should be updated in https://github.com/cfpb/cfgov-refresh/pull/3396/commits/9e021bad68cc8e75a4ea72d5ebd499b0566417d6